services.html

```diff
@@ -156,18 +155,23 @@ <h4>Service Three</h4>
 
         <!-- ========== Service Category 2 ========== -->
 
+        <div style="width: 100%; height: 250px; background-color: lightgrey;">
+          placeholder
+        </div>
+
```

Comment:
@nickfryar Use a placeholder image from [placeholder.com](https://placeholder.com) and center that background image, it'll allow for easier inserting of the image in the upcoming milestone.